main.js

```diff
@@ -51,16 +51,31 @@ var currentActivity;
 var pastActivities = [];
 
 //Add event listeners below
+window.addEventListener(“load”, function() {
```

Comment:
**Nitpick**: Pick either single or double quotes and stick with it. Not anything important, just a consistency thing :P

Reply:
Excellent. It's on our refactor list.